src/Common/src/Interop/Windows/kernel32/Interop.CompareStringOrdinal.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, SetLastError = true, ExactSpelling = true)]
```

Comment:
In some of the P/Invokes you've added, there's a link to the corresponding docs.  In others, there aren't.  What's the reasoning for when one gets a link and one doesn't?

Reply:
Completely honest: where I wrote the P/Invoke originally. I generally copy from my own metaproject if I've already written them- I haven't been consistently adding them for the ones I write from scratch here. I'll be consistent from now on.